pkg/plugins/generic/generic_plugin.go

```diff
@@ -196,28 +201,28 @@ func needDriverCheckVdpaType(state *sriovnetworkv1.SriovNetworkNodeState, driver
 	return false
 }
 
-func tryEnableIommuInKernelArgs() (bool, error) {
-	glog.Info("generic-plugin tryEnableIommuInKernelArgs()")
-	args := [2]string{"intel_iommu=on", "iommu=pt"}
+// trySetKernelParam Tries to add the kernel param via ostree or grubby.
+func trySetKernelParam(kparam string) (bool, error) {
```

Comment:
shall this be renamed to setKernelParam ?
we should never call this one for an already set parameter now.

also now, i dont think we need to return the number of parameters configured in enable-kargs.sh.
so this bit can be removed from script IMO.

Reply:
+1 and same comment wrt param vs args (please use latter terminology).

Reply:
I agree, renamed to setKernelArg